modules/ROOT/pages/dialog-examples.adoc

```diff
@@ -34,7 +34,7 @@ The configuration for the first page of the multipage form contains a descriptio
 
 More specifically:
 
-The `+onChange()+` callback in `+page1Config+` is fired when the checkbox is toggled. It uses `+enable+` and `+disable+` from the dialog instance API to disable and enable the *Next* button. The code uses `+getData()+` from the dialog instance API to get the state of the checkbox called `+anyterms+` (which is `+true+` if checked and `+false+` if unchecked) and choose `+enable()+` or `+disable()+` based on its value. It assigns the correct function to a variable, then calls that variable with the component name `+uniquename+` to toggle the state of the *Next* button.
+The `+onChange()+` callback in `+page1Config+` is fired when the checkbox is toggled. It uses `+enable+` and `+disable+` from the dialog instance API to disable and enable the *Next* button. The code uses `+getData()+` from the dialog instance API to get the state of the checkbox called `+anyterms+` (which is `+true+` if checked and `+false+` if unchecked) and then calls the `+setEnabled+` function with the component name `+uniquename+` and checkbox value to set the enabled state of the *Next* button.
```

Comment:
Not 100% sure but I think this should be from looking at the source code?
```suggestion
The `+onChange()+` callback in `+page1Config+` is fired when the checkbox is toggled. It uses `+setEnabled+` from the dialog instance API to disable and enable the *Next* button. The code uses `+getData()+` from the dialog instance API to get the state of the checkbox called `+anyterms+` (which is `+true+` if checked and `+false+` if unchecked) and then calls the `+setEnabled+` function with the component name `+uniquename+` and checkbox value to set the enabled state of the *Next* button.
```

Reply:
Oops, yep definitely missed that reference and that looks correct thanks!